README.md

```diff
@@ -1 +1,34 @@
-# To_Do_list_interactive-
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Personal Portfolio Page
+
+> This is the Awesome books: plain JavaScript with objects that shows of my accomplishments as a software developer.
+
+
+
+## Built With
+
+- HTML, CSS and JavaScript
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
```

Comment:
- [ ] Kinldy make sure you provide descriptive and easy-to-understand instructions on how to run the project locally.

`hint: You may say: To clone the repos run in the terminal git clone <REPO_LINK>`
`To navigate over it : cd <DIRECTORY_NAME>`
`You may also consider not forgetting about the dependencies insatllation` :smile: 
`...`